packages/components/src/components/Link/Link.tsx

```diff
@@ -239,6 +249,7 @@ export const Link: FC<LinkProps> = ({
 
   const pressableProps: PressableProps = {
     onPress: _onPress,
+    hitSlop: 7,
```

Comment:
Would love to know the origin of this naming 🤣

Reply:
No idea, but [found this searching it...](https://www.reddit.com/r/reactnative/comments/ib1euo/i_made_a_hitslop_hat/)